site/content/docs/5.1/components/scrollspy.md

```diff
@@ -168,20 +168,34 @@ Scrollspy also works with nested `.nav`s. If a nested `.nav` is `.active`, its p
 </nav>
 
 <div data-bs-spy="scroll" data-bs-target="#navbar-example3" data-bs-offset="0" tabindex="0">
-  <h4 id="item-1">Item 1</h4>
-  <p>...</p>
-  <h5 id="item-1-1">Item 1-1</h5>
-  <p>...</p>
-  <h5 id="item-1-2">Item 1-2</h5>
-  <p>...</p>
-  <h4 id="item-2">Item 2</h4>
-  <p>...</p>
-  <h4 id="item-3">Item 3</h4>
-  <p>...</p>
-  <h5 id="item-3-1">Item 3-1</h5>
-  <p>...</p>
-  <h5 id="item-3-2">Item 3-2</h5>
-  <p>...</p>
+  <div id="item-1">
```

Comment:
I can't see why we needed these surrounding divs ? And why only around the nested nav example ?

Could you light me up about this please @GeoSot ? 😄

Reply:
Yes of course.
Scrollspy Observer checks if a specific target-element is visible inside the root (wrapper-element).
In order to keep the functionality closer to the old implementation, If  the element gets out of visible wrapper area, is marked as de-activated.
 
So for big sections and specialy in the case a section is bigger than its wrapper (scrollspy) height, we may fall in the case where the heading element, will not be visible and the corresponding  menu-item will be de-activated.  

The previous is not the standard case, and in addition, many developers already use it this way (observables are the sections , not the headings) 

As of the above, I don't find it a bad idea to include the alternative approach, helping newcomers getting alternative implementation ideas

Reply:
Thanks a lot for for this explanation !